lighthouse-core/audits/offline-start-url.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+
+class OfflineStartUrl extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'offline-start-url',
+      title: 'start_url responds with a 200 when offline',
```

Comment:
should we be trying to `i18n` new strings in general? feels like we should

unrelated: `start_url` feels so weird in the title like that haha

Reply:
> should we be trying to i18n new strings in general? feels like we should

I like doing them with everyone all at once so I can avoid responsibility for them, but should we do them as we go?

> unrelated: `start_url` feels so weird in the title like that haha

haha, I felt the same, but it felt worse to just have them be words. "Start URL responds with a 200..."

Reply:
Maybe "Service Worker's start_url responds with a 200"?

Reply:
> Maybe "Service Worker's start_url responds with a 200"?

yeah, it's not the service worker's `start_url`, but the service worker needs to respond to the request for the `start_url`...